FirebaseMLModelDownloader/Sources/ModelInfoRetriever.swift

```diff
@@ -265,6 +251,7 @@ class ModelInfoRetriever {
                     .missingModelHash)))
                 return
               }
+              /// Ensure that there is local model info on device with matching hash.
```

Comment:
Nit: 2 `//` only

Reply:
Good catch. Updated everywhere not a property/method/class description to be `//`.